### DISCUSSION TITLE
Primefaces Extensions Showcase not working.

### DISCUSSION THREAD

**Discussion by Cprusinowski** [primefaces/community#1847] [PrimeFaces] [answered]
Am I the only one with _HTTP ERROR 503 Service Unavailable_ when enter to https://www.primefaces.org/showcase-ext/views/home.jsf?

**Comment by melloware** [comment, marked as answer]
Yep it is down.  PrimeTek is trying to upgrade to JDK 11 and deploy the 14.0.0 Showcases.

**Reply by Cprusinowski** [reply]
Thanks for info.
